app/controllers/api/concerns/search/permit_applications.rb

```diff
@@ -28,11 +27,7 @@ def permit_application_search_params
       :query,
       :page,
       :per_page,
-      filters: {
-        status: [],
-        template_version_id: :template_version_id,
-        requirement_template_id: :requirement_template_id,
-      },
+      filters: [:requirement_template_id, :template_version_id, { status: [] }],
```

Comment:
is the brackets around status intentional

Reply:
yes, this is the only way I can get this to work (it's weird in rails 7.1 i think)